.gitignore

```diff
@@ -0,0 +1 @@
+.vscode
```

Comment:
Don't add this to the repo's .gitignore. Instead, add this into your local `.git/info/exclude` file